<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Medician2</class>
 <widget class="QWidget" name="Medician2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="apply_pwz_push_button">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwz_line_edit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Insert your PWZ</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_box_patient">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>310</y>
     <width>241</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="display_text_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>521</width>
     <height>181</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_visits">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Visits</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_patient">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>280</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_ref">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>460</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add referral</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_presc">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>500</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add prescription</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_desc">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>460</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add description</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_push_button">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>590</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="display_history_text_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>521</width>
     <height>341</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_history">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Medical history</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_appointment">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>390</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Appointment</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_box_appointments">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>420</y>
     <width>241</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_edit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>500</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_patient_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>110</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select date</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_box_patient_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>241</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
